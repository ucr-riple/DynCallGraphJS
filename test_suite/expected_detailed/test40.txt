{
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test40.js:1:1:12:5)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test40.js:1:2:12:2)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test40.js:10:3:10:26)': [
        'shift (Native)(/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test40.js:10:3:10:26)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test40.js:10:3:10:28)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test40.js:3:5:5:6)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test40.js:11:3:11:23)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test40.js:6:5:8:6)'
    ]
}