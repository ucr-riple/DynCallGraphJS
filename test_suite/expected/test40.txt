{ '(test40.js:1:1:12:5)':
   [ '(test40.js:1:2:12:2)' ],
  '(test40.js:10:3:10:26)': [ 'shift (Native)' ],
  '(test40.js:10:3:10:28)':
   [ '(test40.js:3:5:5:6)' ],
  '(test40.js:11:3:11:23)':
   [ '(test40.js:6:5:8:6)' ] }